### DISCUSSION TITLE
High quality renderer for pictures

### DISCUSSION THREAD

**Discussion by kheinz57** [bevyengine/bevy#10242] [Q&A] [answered]
Hi all. I am trying to build kind of a virtuell museum or call it 3d picture viewer. So you can walk or fly around and see paintings. Mainly as a learning project of Rust and bevy. And as an example to show art. 
But the quality of the images seems to be pretty poor. Even when I come prettx close. Currently by using the bevy_flycam. 
Would be nice to improve the quality or at least have to chance to switch to a complete 2d view.
Any ideas are welcome

**Comment by DGriffin91** [comment]
Can you share a screenshot showing the quality issues? You'll probably want to generate mipmaps, use 16x anisotropic filtering, and set a bit of a negative mip bias.

**Reply by kheinz57** [reply]
Thanks for the answer. Since I am pretty new to the rendering and gaming stuff I have read more on your mentioned topics. It might as well be depending on tghe light. As of now I have only an ambient light in White with 0.8 brightness. 
Here is the screenshot as coparison between left the bevy and right the oSX preview output. 
 
![grafik](https://github.com/bevyengine/bevy/assets/281508/190dc41a-3e98-4f9a-9047-5a6425e2612b)

**Reply by tim-blackbird** [reply, marked as answer]
The quality looks fine other than the brightness and contrast being off.

Set `unlit` to `true` on the material if you want to see the paintings as-is without any lighting applied.

You could also try placing a spotlight by the paintings.

**Reply by kheinz57** [reply]
Thanks, that solved it. 
Spotlight was something I tried but still failed. Sounds like it will be in the end the better solution. But I guess when really zooming in on one painting te unlit seems like the best solution. 
Will try to make the application at least a bit workig and will then publish on github. Since I now do have the confidence it will show the paintings correctly so I can preceed.

**Reply by kheinz57** [reply]
I have published it now. Very early version of course: https://github.com/kheinz57/theartgame
